nnvm/python/nnvm/frontend/keras.py

```diff
@@ -131,6 +131,14 @@ def _convert_dense(insym, keras_layer, symtab):
     if keras_layer.use_bias:
         params['use_bias'] = True
         params['bias'] = symtab.new_const(weightList[1])
+    input_shape = keras_layer.input_shape
+    input_dim = len(input_shape)
+    # In case of RNN dense, input shape will be (1, 1, n)
+    if input_dim > 2:
+        input_shape = tuple(dim if dim else 1 for dim in _as_list(input_shape)[0])
+        if input_dim != 3 and input_shape[0] != input_shape[1] != 1:
```

Comment:
I think this check should be
```suggestion
        if input_dim != 3 or input_shape[0] != 1 or input_shape[1] != 1:
```
or
```suggestion
        if not (input_dim == 3 and input_shape[0] == input_shape[1] == 1):
```